tests/unit/array/sort-test.js

```diff
@@ -0,0 +1,83 @@
+import { sort } from 'ember-awesome-macros/array';
```

Comment:
ok, well in that case what i've written as unit tests are in fact integration tests...but half of what's currently in the integration tests file are unit tests :P

I'll move mine over to the existing test file, and update the PR